lib/databases/mongodb.js

```diff
@@ -210,7 +218,7 @@ _.extend(MongoSessionStore.prototype, {
 
     if (!this.options.ignoreConcurrency) sess._hash = new ObjectID().toString();
 
-    this.sessions.update(query, sess, { upsert: true, safe: true }, function(err, modifiedCount) {
+    this.sessions.updateOne(query, { $set: sess }, { upsert: true }, function(err, modifiedCount) {
```

Comment:
Can you write it backwards compatible?
i.e. if typeof this.sessions.updateOne === 'function'  ? updateOne : update